packages/server-wallet/src/objectives/close-channel.ts

```diff
@@ -16,7 +16,11 @@ export class CloseChannelObjective {
           {
             type: 'CloseChannel',
             participants: [],
-            data: {targetChannelId: channelId, fundingStrategy: channel.fundingStrategy},
+            data: {
+              targetChannelId: channelId,
+              fundingStrategy: channel.fundingStrategy,
+              txSubmitterOder: [1, 0],
```

Comment:
I suppose you are asserting (paraphrasing from #3205)
> For now, the `txSubmitterOrder` is always set to `[1, 0]`

⚠️ It is spelled wrong throughout this PR.

```suggestion
              txSubmitterOrder: [1, 0],
```

Reply:
Good catch! c39a36102